Userland/Applications/Browser/DownloadWidget.cpp

```diff
@@ -80,7 +83,10 @@ DownloadWidget::DownloadWidget(const URL& url)
     m_progress_label->set_text_alignment(Gfx::TextAlignment::CenterLeft);
     m_progress_label->set_fixed_height(16);
 
-    auto& destination_label = add<GUI::Label>(String::formatted("To: {}", m_destination_path).release_value_but_fixme_should_propagate_errors());
+    DeprecatedString m_to_label_path = m_destination_path;
+    m_to_label_path = m_to_label_path.substring((size_t)0, m_to_label_path.find_last('/').value_or(0));
```

Comment:
It looks like this comment got lost perhaps? Or is there a reason why we need that C-style cast and the string-operation instead of calling LexicalPath?

How about this, to replace the two previous lines:
```suggestion
    auto destination_label_path = LexicalPath(m_destination_path).dirname();
```
(Plus a `#include <AK/LexicalPath.h>` at the top.)